cabal-testsuite/PackageTests/GhcPkgGuess/SymlinkVersion/setup.test.hs

```diff
@@ -1,11 +1,12 @@
 import Test.Cabal.Prelude
 -- TODO: Enable this test on Windows
-main = setupAndCabalTest $ do
-    skipIfWindows
-    withSymlink "bin/ghc-7.10" "ghc" $ do
+main = setupAndCabalTest $ expectBrokenIf isWindows 10179 $ do
+    (if isWindows
+        then withSymlink "bin/ghc-7.10.bat" "ghc.bat"
+        else withSymlink "bin/ghc-7.10" "ghc") $ do
         env <- getTestEnv
         let cwd = testCurrentDir env
         ghc_path <- programPathM ghcProgram
         r <- withEnv [("WITH_GHC", Just ghc_path)]
-           . fails $ setup' "configure" ["-w", cwd </> "ghc"]
+           . fails $ setup' "configure" ["-w", (if isWindows then (<.> "bat") else id) $ cwd </> "ghc"]
```

Comment:
That `if isWindows` clause occurs enough that I wonder if it should be in `Test.Cabal.Prelude`. Might be future work, though.

Reply:
This is gone now. See https://github.com/haskell/cabal/issues/10179#issuecomment-2215475067